R/pkg/R/functions.R

```diff
@@ -1730,24 +1730,82 @@ setMethod("toRadians",
 
 #' to_date
 #'
-#' Converts the column into DateType.
+#' Converts the column into DateType, optionally with a format. Returns null if input is invalid.
```

Comment:
`Returns null if input is invalid.` - seems like this could confuse things since the sql function we don't actually NULL - would the column contains NULL value as a result? If so we should explain that here. Also, I'm pretty sure null in JVM gets turn into `NA` in R, might want to check for this case.